src/components/related_items_outfit/riProductCard.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Price from './Price';
+import Image from './Image';
+
+export default function ProductCard() {
+  return (
+    <div>
+      <Image />
+      {/* Product Category: */}
+      <p>Jackets</p>
+      {/* Product Name: */}
+      <p>Camo Onesie</p>
+      {/* Price: */}
+      <Price price={140} salePrice={120} />
+      {/* Star Rating */}
```

Comment:
See this stub for the star -- feel free to import the Stars component from the `shared` folder whenever you submit your next PR